db/migrate/20180706134018_drop_repo_events.rb

```diff
@@ -0,0 +1,9 @@
+class DropRepoEvents < ActiveRecord::Migration[5.1]
+  def up
+    drop_table :repo_events
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration, 'cannnot restore table data'
```

Comment:
is this created by the generator or do you have to add it?

Reply:
You have to add it